### DISCUSSION TITLE
Framework compatibility

### DISCUSSION THREAD

**Discussion by victorsolaya** [microsoft/omnichannel-chat-sdk#218] [Q&A] [answered]
Hi, 

I have seen that your examples are referring always to react. Is there any way to fit this with Angular, Vue.
Has anybody succeded on this?

Thank you,

**Comment by xTEddie** [comment, marked as answer]
Yes, we've had customers using `omnichannel-chat-sdk` on Angular. `omnichannel-chat-sdk` does not have dependencies on any web frameworks as long as `client-side rendering` is being used.
